api/v0/ingest/schema/gen.go

```diff
@@ -70,16 +70,21 @@ func main() {
 		// ID is a multihash of IndexID+Provider+Previous
 		schema.SpawnStructField("ID", "Bytes", false, false),
 		schema.SpawnStructField("IndexID", "Link_Index", false, false),
-		schema.SpawnStructField("Previous", "Link_Advertisement", true, false),
+		// Empty bytes for the first advertisement
+		schema.SpawnStructField("PreviousID", "Bytes", false, false),
```

Comment:
why bytes rather than an optional link?

Reply:
When a `legSubscriber`sees an advertisement we don't want it to lfollow advertisement links, as it is only interested in the data. Indices point to their previous index so from an advertisement we can directly fetch all our pending updates.

This previousID will be used for the paginated ingestion protocol, or for indexers to request individual advertisements from a provider, thus the bytes. This is also the most convenient structure to use the `stopAt` selector.

Let me know if something doesn't make sense.